src/main/resources/application-local-dev.yaml

```diff
@@ -24,6 +26,25 @@ digdir:
       feature-enabled: false
 
 eidas:
+  oidc-server:
+    issuer: http://localhost:8080
+    clients:
```

Comment:
legg vel inn ein ny klient her etterkvart for eidas

Reply:
det er vel idporten som er klient her?